website/docs/ecosystem/organizations.md

```diff
@@ -0,0 +1,19 @@
+---
+id: organizations
+title: Organizations
+---
+
+# Organizations
+
+Explore all the organizations and their relevant GitHub repositories within the decentralized Nervos ecosystem.
+
+| Org name       | Github repo                      | Major contents                                    |
+| -------------- | -------------------------------- | ------------------------------------------------- |
+| Nervos Network | https://github.com/nervosnetwork | CKB-VM, RFCs, Moleculec-es                        |
+| Cell Studio    | https://github.com/ckb-cell      | RGB++                                             |
+| Spore Protocol | https://github.com/sporeprotocol | Spore protocol                                    |
+| Cryptape       | https://github.com/cryptape      | CKB Node Probe, Omnilock, CoBuild, Anyone-can-pay |
+| Magickbase     | https://github.com/Magickbase    | CKB Explorer, Neuron, CKB Testnet Faucet          |
+| Nervina-labs   | https://github.com/nervina-labs  | JoyID, CoTA                                       |
+| CKB-JS         | https://github.com/ckb-js        | Lumos, Kuai, CKB-SDK-JS                           |
+| .bit           | https://github.com/DIDHQ         | DID, Voty                                         |
```

Comment:
should be https://github.com/dotbitHQ ? the content can be  did-contracts , dotbit.js